File: android/app/src/main/java/com/example/flutterembedded/flutter/FlutterChannelManager.kt
package com.example.flutterembedded.flutter

import android.content.Context
import android.util.Log
import io.flutter.embedding.engine.FlutterEngine
import io.flutter.plugin.common.MethodChannel

class FlutterChannelManager(private val engine: FlutterEngine,private val context: Context?) {

    init {
        initTestChannel()
    }

    private fun initTestChannel(){

        MethodChannel(
                engine.dartExecutor.binaryMessenger,
                "TestChannel"
        ).setMethodCallHandler { call, result ->
            Log.i("FlutterModule", call.method)
            result.success(null)
        }

    }

}

File: ios/IOSFlutterEmbbeded/FlutterChannelManager.swift
//
//  FlutterChannelManager.swift
//  IOSFlutterEmbbeded
//
//  Created by Rafael Barbosa on 24/02/21.
//

import Flutter;
import Foundation

class FlutterChannelManager{
        
    let engine:FlutterEngine
    
    init(engine: FlutterEngine) {
        self.engine = engine
        self.setup()
    }
    
    private func setup(){
        
        FlutterMethodChannel(
            name: "TestChannel",
            binaryMessenger: engine.binaryMessenger
        ).setMethodCallHandler({(call: FlutterMethodCall, result: @escaping FlutterResult) -> Void in
            
            print(call.method);
            
        });
    }
}
